exercises/clock/clock_test.rb

```diff
@@ -145,4 +145,81 @@ def test_wrap_around_backwards
     clock = Clock.at(0, 30) - 60
     assert_equal '23:30', clock.to_s
   end
+
+  def test_clocks_with_same_time
+    skip
+    clock1 = Clock.at(15, 37)
+    clock2 = Clock.at(15, 37)
+    assert_equal clock1, clock2
+  end
+
+  def test_clocks_a_minute_apart
+    skip
+    clock1 = Clock.at(15, 36)
+    clock2 = Clock.at(15, 37)
+    refute_equal clock1, clock2
+  end
+
+  def test_clocks_an_hour_apart
+    skip
+    clock1 = Clock.at(14, 37)
+    clock2 = Clock.at(15, 37)
+    refute_equal clock1, clock2
+  end
+
+  def test_clocks_with_hour_overflow
+    skip
+    clock1 = Clock.at(10, 37)
+    clock2 = Clock.at(10, 37) + days(1) + hours(1)
+    refute_equal clock1, clock2
+  end
+
+  def test_clocks_with_hour_overflow_by_several_days
+    skip
+    clock1 = Clock.at(3, 11)
+    clock2 = Clock.at(0, 11) + hours(99)
+    assert_equal clock1, clock2
```

Comment:
With the current example implementation, no, because the `Clock.at(0, 11+hours(99))` will fail before you get to that point.  It would fail with an "Argument Error: argument out of range".

With a different implementation, definitely.

The tests as they stand works for the current example.rb file.  The example.rb is probably relying too much on Ruby built in time.  Coincidentally, it is implemented without dates, and so fits.